### DISCUSSION TITLE
Tooltips on buttonsList

### DISCUSSION THREAD

**Discussion by tomitomas** [add2cal/add-to-calendar-button#500] [Ideas / Requests]
Hi, 

thanks for that great product ! 

Ever think about getting a tooltip hover the icon when using the `buttonsList` option ? 
Might be great to get the (custom) label of every options in a tooltip when the user makes a mouse hover each icon ...? 

Any thoughts ? 

Cheers,

**Comment by jekuer** [comment]
What should the tooltip say? A custom message or the calendar name?

**Comment by tomitomas** [comment]
yep, or whatever the user setup in the `options` item 

`options=\"'iCal|iCal is ok for me','Google|Add to Google Cal','Outlook.com|Use Outlook instead'\"`

will give : 
![image](https://github.com/add2cal/add-to-calendar-button/assets/12429858/61f543aa-109b-48b6-89be-0f1f4a390cd7)

**Comment by tomitomas** [comment]
the infos looks to be already available in the `aria-label` tag --> only needs to be duplicate as `title` (!?)

but with a small  "issue" as it takes only the first value. here the 3 icons have the same aria-label :
![image](https://github.com/add2cal/add-to-calendar-button/assets/12429858/fe2daf43-a286-41b1-976d-7fcedfb4fb4d)

**Reply by jekuer** [reply]
Thanks for this catch - will be fixed soon.

Regarding the Tooltip: So, are you only thinking about having the title attribute set - or are you talking about a real fancy tooltip?

**Reply by tomitomas** [reply]
title attribute can be a first step :)
fancy tooltip can be a v2 :)

you already offer several options : that could one more --> displayTooltip : standard(title) | fancy
